.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+# Based on https://www.eliostruyf.com/publishing-vscode-extensions-github-actions/
```

Comment:
Please don't mix unrelated changes with a bug fix.

Reply:
Yeah, sorry about that.
I removed the commits that I was not planning to merge.